app/api/auth.py

```diff
@@ -138,6 +140,24 @@ def login_user(provider, auth_code):
             'client_id': provider_class.get_client_id(),
             'client_secret': provider_class.get_client_secret()
         }
+        if not payload['client_id'] and payload['client_secret']:
+            return make_response(jsonify(message='Application ID and Secret key not found'), 403)
+        twitter_blueprint = make_twitter_blueprint(api_key=payload['client_id'], api_secret=payload['client_secret'])
+        app.register_blueprint(twitter_blueprint, url_prefix='/twitter_login')
+        if not twitter.authorized:
+            return redirect(url_for('twitter.login'))
+        account_info = twitter.get('account/settings.json')
+        if account_info.ok:
+            account_info_json = account_info.json
+        user = User()
+        user.first_name = account_info_json['name']
+        user.twitter_id = account_info_json['screen_name']
```

Comment:
@mrsaicharan1 you are using a `db.BigInteger` column to store a string(`screen_name`). Is this valid?